bgcval2/functions/circulation.py

```diff
@@ -177,9 +257,87 @@ def TwentySixNorth(nc,keys,**kwargs):
     for z in range(e3v_AMOC26N.shape[0] -2, 1, -1): # add from the bottom up
         atlmoc[z, :] = atlmoc[z+1, :] + atlmoc[z, :]
     return atlmoc
-       
+
+
+def twentysixnorth025(nc,keys,**kwargs):
+    """
+    This function loads the AMOC array that is used for eORCA025
+
+    nc: a netcdf openned as a dataset.
+    keys: a list of keys to use in this function.
+
+    """
+    areafile = get_kwarg_file(kwargs, 'areafile')
+    maskname = kwargs.get('maskname', 'tmask')
+    grid = kwargs.get('grid', 'eORCA025')
+
+    if not loadedArea:
+        loadDataMask(areafile, maskname, grid)
+
+    if grid != 'eORCA025':
+        assert 0
+    
+    latslice26N = eORCA025_latslice26Nnm
+    lonslice26N = eORCA025_lonslice26Nnm
+    vo =  np.ma.array(nc.variables[keys[0]][..., latslice26N, lonslice26N]) # #vo in m/s
+    thkcello = np.ma.array(nc.variables['thkcello'][..., latslice26N, lonslice26N]) # #thickness
+    depths = np.ma.cumsum(thkcello, axis=1)
+
+    depths = np.ma.masked_where(thkcello.mask + np.abs(depths)<500., depths) # masked above 500m depth.
+
+    e1v = e1v_AMOC26N[:,None, :, :]
+    flux = vo * depths * e1v_AMOC26N[:,None, :, :]/1.E06
+    
+    moc=np.ma.zeros_like(flux)
+    np.ma.cumsum(flux[:,::-1], axis=1, out=moc ) # sum floor to surface
```

Comment:
np.cumsum is a bit weird like that. It takes a `out` argument and sends the result there!

Reply:
but you're not using it correctly: https://numpy.org/doc/stable/reference/generated/numpy.ma.cumsum.html - you either call it as a method on your array so `arr = np.ma.array(...); arr.cumsum()` or you define a new variable (or the same) `arr = np.ma.cumsum(arr, axis=axis, ...)`

Reply:
That's the `out` field.

Reply:
But this is does work. The output is sent to `out`, and flux is unchanged. That's what we want. 

An alternative way to do this would be:
```
    moc = np.ma.cumsum(flux[:,::-1], axis=1) # sum floor to surface
```
or:
```
    moc = flux[:,::-1]cumsum(axis=1) # sum floor to surface
```